packages/augur-ui/src/modules/create-market/components/form.tsx

```diff
@@ -619,11 +619,12 @@ export default class Form extends React.Component<FormProps, FormState> {
               )}
               {mainContent === REVIEW && <Review />}
               {mainContent === TEMPLATE_PICKER && <TemplatePicker />}
-              {mainContent === SUB_CATEGORIES && <SubCategories />}
+              {mainContent === SUB_CATEGORIES && <SubCategories nextPage={this.nextPage}/>}
```

Comment:
its weird that sub category page goes to next page automatically but the market type page or template picker one doesn't

Reply:
I'd like the market type page to auto page as well. Template picker does auto-page